cves/kernel/CVE-2019-15807.yml

```diff
@@ -373,8 +387,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    While the order of things were not out of the ordinary, the initial callbacks
```

Comment:
There was weird spacing here too. It's just that 'callbacks' is on a new line :)